### PR TITLE
addProject

### PR DESCRIPTION
**Pull request by KovalenkoOlga**
Add frontend and backend app

### CODE REVIEW

File: Server/.env [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PORT=5000
+DB_USER=gdcn
+DB_HOST=localhost
+DB_SCHEME=goodcoin
+DB_PASSWORD=gdcn
+DB_POST=5432
```
Comment (mkostrikin) [inline Server/.env]:
```suggestion
DB_PORT=5432
```

File: Server/main/db.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const { Pool } = require('pg')
+
+const pool = new Pool({
+  user: process.env.DB_USER,
+  host: process.env.DB_HOST,
+  database: process.env.DB_SCHEME,
+  password: process.env.DB_PASSWORD?process.env.DB_PASSWORD:'',
+  post: process.env.DB_POST
```
Comment (mkostrikin) [inline Server/main/db.js]:
```suggestion
  post: process.env.DB_PORT

```

File: Client/src/context_state_config.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import Context from './utils/context';
+import Core from "./component/core";
+
+const ContextState = () => {
+    const getApiUrl = () => {
+        return process.env.REACT_APP_API_ENDPOINT
+    }
+
+    const getRecordNumber = () => {
+        const rowNum = process.env.REACT_APP_RECORD_NUMBER_TO_DISPLAY
```
Comment (n0th1ng-else) [inline Client/src/context_state_config.js]:
```suggestion
        const rowNum = Number(process.env.REACT_APP_RECORD_NUMBER_TO_DISPLAY);
```

File: Client/src/context_state_config.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        return !rowNum || +rowNum > 100 ? 20 : rowNum
```
Comment (n0th1ng-else) [inline Client/src/context_state_config.js]:
```suggestion
        return !rowNum || rowNum > 100 ? 20 : rowNum
```

File: Client/public/index.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (n0th1ng-else) [inline Client/public/index.html:5]: how this PUBLIC_URL populates?

File: Client/src/component/core.js [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+                console.log(err)
+            })
+    };
+
+    onFinishFailed = errorInfo => {
+        console.log('Failed:', errorInfo);
+    };
+
+
+    layout = {
```
Comment (n0th1ng-else) [inline Client/src/component/core.js]: move class properties above the methods

**Comment by n0th1ng-else** [comment]
~please mark all Class properties either public or private so we can distinguish if it being used somewhere or not~
nevermind, it is my Typescript mind. we good

**Review by n0th1ng-else — APPROVED** [review]

**Review by mkostrikin — APPROVED** [review]

**Pull request merged**